### PR TITLE
Pull GameViewController to refresh

### PR DESCRIPTION
**Pull request by HugoBernal**
# ISSUE #28 

# In this PR
Added refreshing roles and tableView by pulling the table in GameViewController.

# Evidence
simulator screen shot - iphone 8 plus - 2018-01-23 at 22 54 26

### CODE REVIEW

File: MAFIA/Views/Controllers/Game/GameViewController.swift [hunk 1]
```diff
@@ -27,6 +27,7 @@ class GameViewController: UIViewController {
             updateGameUI()
         }
     }
+    private var pullToRefresh: UIRefreshControl!
```
Comment (lramirezsuarez) [inline MAFIA/Views/Controllers/Game/GameViewController.swift]: It's better to put the global variables together in the top of the view controller with the rest of the global variables and outlets.

Comment (lramirezsuarez) [reply MAFIA/Views/Controllers/Game/GameViewController.swift]: It was in the global variables, well done 😂

Comment (lramirezsuarez) [reply MAFIA/Views/Controllers/Game/GameViewController.swift]: It was in the global variables, well done 😂

**Comment by santicarmo31** [comment]
Great work Hugono, Continue learning you are going on the correct way !!

**Pull request merged**